### ISSUE TITLE
 Fatal error: Declaration of WCPay\Internal\DependencyManagement\DelegateContainer\LegacyContainer::has($id) must be compatible with Psr\Container\ContainerInterface::has(string $id): bool

### ISSUE THREAD

**Issue by Lenny4** [Automattic/woocommerce-payments#7488]
With PHP 8.2.4 I got error

```
 Fatal error: Declaration of WCPay\Internal\DependencyManagement\DelegateContainer\LegacyContainer::has($id) must be compatible with Psr\Container\ContainerInterface::has(string $id): bool
```
In 3 files:
- woocommerce-payments/src/Container.php
- woocommerce-payments/src/Internal/DependencyManagement/DelegateContainer/LegacyContainer.php
- woocommerce-payments/src/Internal/DependencyManagement/DelegateContainer/WooContainer.php

**Comment by nicdwilson** [comment]
7169346-zd-a8c
7160929-zd-a8c
7163053-zd-a8c

**Comment by nicdwilson** [comment]
Conflicts identified so far
- Divi Product Carousel

**Comment by danielk-am** [comment]
7172522-zd-a8c
7162700-zd-a8c

**Comment by davidhealey** [comment]
Seems to also conflict with BackWPup - the error occurred for me after updating that plugin and disabling that plugin got rid of the issue on my site.

**Comment by jtobiesen** [comment]
7189692-zd-a8c

**Comment by htdat** [comment]
<strike>Sorry everyone. WooPayments 6.6.1 has been released to aim this issue (and others).
I am closing this issue. But needed, please create a new issue. </strike>

**Comment by htdat** [comment]
Sorry again :( I mislead the information that 6.6.1 included the fix for this issue. Re-opening it now.

**Comment by EjayhanFernandes** [comment]
7192410-zd-a8c - Deactivating BackWPup resolved the issue for now.

**Comment by NicolasMarty11** [comment]
Same problem here. Disabling BackWPup resolve the issue

**Comment by daniyalahmadk** [comment]
7193375-zd-a8c. deactivating BackWPup resolved the issue

**Comment by RadoslavGeorgiev** [comment]
Fixed through #7525 in 6.6.2
